org_chart.js

```diff
@@ -1,70 +1,43 @@
-let elon = {
-  name: "Elon Musk",
-  email: "elon.musk@musky.com",
-  position: "Boss",
+function Persons (name, email, position, project, gender) {
```

Comment:
I wouldn't use a plural for the name, as this constructor creates ONE person object